src/makers/dragino/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 27, 
```

Comment:
There is an extra space to remove here. You can find the lint errors by running `yarn lint`.
```
hotspot-app/src/makers/dragino/index.ts
  7:10  error  Delete `·`  prettier/prettier
```

Reply:
Just fix. Thanks!